sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducerAsyncClient.java

```diff
@@ -425,10 +427,11 @@ public String getEventHubName() {
         return getSendLink(batch.getPartitionId())
             .flatMap(link -> {
                 if (isTracingEnabled) {
-                    Context entityContext = finalSharedContext.addData(ENTITY_PATH_KEY, link.getEntityPath());
+                    Context entityContext = finalSharedContext.addData(ENTITY_PATH_KEY, link.getEntityPath())
```

Comment:
nit: It is a lot easier to understand all the method invocations by formatting it as:
```java
finalSharedContext.addData(ENTITY_PATH_KEY, link.getEntityPath())
    .addData(HOST_NAME_KEY, link.getHostname())
    .addData(AZ_NAMESPACE_KEY, AZ_NAMESPACE_VALUE);
```

Same comment as the ones below.